src/output-json-stats.c

```diff
@@ -226,8 +227,13 @@ json_t *StatsToJSON(const StatsTable *st, uint8_t flags)
     uint32_t u = 0;
     if (flags & JSON_STATS_TOTALS) {
         for (u = 0; u < st->nstats; u++) {
+            bool is_exception_policy_counter = false;
             if (st->stats[u].name == NULL)
                 continue;
+            if (strstr(st->stats[u].name, "exception_policy") != NULL) {
+                is_exception_policy_counter = true;
+            }
```

Comment:
This doens't look like the best way to do this. (This and what follows). Moreover, with the introduction of summary counters for App Layer error exception policy counters, maybe this isn't the best option? Not sure yet...